Rename getPrevalence to addPrevalence and getPrevalentAbundance to addPrevalentAbundance

theotimeee opened the pull request:
This PR makes the following naming modifications : 

- `getPrevalence` --> `addPrevalence`
- `getPrevalentAbundance` --> `addPrevalentAbundance`

Commit:
rename getPrevalentAbundance to addPrevalentAbundance

Commit:
rename getPrevalence to addPrevalence

Commit:
fix duplicate alias warning by renaming getPrevalence R and man files to addPrevalence

Commit:
merge devel

Merge branch 'devel' of https://github.com/microbiome/mia into rename_prevalence_functions

# Conflicts:
#	R/deprecate.R
#	man/deprecate.Rd

theotimeee committed:
Update DESCRIPTION

Commit:
fix deprecate doc

Commit:
Merge branch 'rename_prevalence_functions' of https://github.com/microbiome/mia into rename_prevalence_functions

TuomasBorman commented:
getPrevalence and getPrevalentAbundance return vectors. They do not add results to TreeSE. That is why they should be kept as get*

Commit:
merge devel

Merge branch 'devel' of https://github.com/microbiome/mia into rename_prevalence_functions

# Conflicts:
#	DESCRIPTION
#	NAMESPACE
#	R/addPrevalence.R
#	man/addPrevalence.Rd

Commit:
fix example addPrevalence file
